README.md

```diff
@@ -91,16 +93,44 @@ $ yarn prepare
 $ yarn dev
 ```
 
-## Testing a production build
+## Running development (moderation panel with our local db) 
```

Comment:
I think we should make these optional sections accordion menus
- mod panel
- running production mode to test a build